docker/http-to-bucket/app.rb

```diff
@@ -82,6 +82,8 @@ def upload_response(project_id, bucket_name, object_name, http_response)
 
     request.logger.info "Parameters are valid"
 
+    clean(request)
```

Comment:
It's the response that needs to be cleaned, as part of `upload_response()`, probably between lines 50 and 51, e.g. `clean(body_json)`.